README.md

```diff
@@ -36,15 +36,15 @@ Join gridstack.js on Slack: https://gridstackjs.troolee.com
 - [Changes](#changes)
 - [The Team](#the-team)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- END doctoc generated TOC please keep comment here to allow auto update 
 
 
 gridstack.js News
```

Comment:
probably best not to talk about 1.0 release 2 years later IMO.... better to release couple times a year.